scripts/generate_node_keys.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+docker pull subspacelabs/subspace-node:latest
+node_count=${1}
+output_file=${2}
+if [ -s "${output_file}" ]; then
+    echo "Node keys exists..."
+    exit 0
+fi
+ips=$(echo "$NODE_PUBLIC_IPS" | awk -F, '{for(i=1;i<=NF;i++) print $i}')
+ips=( ${ips} )
+echo -n > "${output_file}"
+echo -n > /tmp/additionl_args.txt
+echo "Generating node keys..."
+for (( i = 0; i < node_count; i++ )); do
+    docker run --rm subspacelabs/subspace-node key generate-node-key &> /tmp/log.txt
```

Comment:
I'd store it in a variable rather than writing to the file without deleting it afterwards.